src/chainparams.cpp

```diff
@@ -1275,9 +1360,7 @@ class CLiquidV1TestParams : public CLiquidV1Params {
             consensus.vDeployments[Consensus::DEPLOYMENT_DYNA_FED].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
         }
 
-        if (args.IsArgSet("-con_taproot_signal_start")) {
-            consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = gArgs.GetArg("-con_taproot_signal_start", 0);
-        }
+        UpdateElementsActivationParametersFromArgs(consensus, args);
```

Comment:
Why didn't you use the same method name as upstream? It's a trade-off between not getting conflicts when they make trivial changes to that method and not getting conflicts when they make critical changes to that method :sweat_smile:

Reply:
Because even though it is the same idea, it does more things than the method upstream, it uses more arguments (Period and Threshold) and makes them optional. Also in upstream it is a method of CRegTestParams and here we also need to call it from liquidv1test